chain/epoch_manager/src/lib.rs

```diff
@@ -329,24 +343,16 @@ impl EpochManager {
         let epoch_info = self.get_epoch_info(last_block_info.epoch_id())?.clone();
         let next_epoch_id = self.get_next_epoch_id(last_block_hash)?;
         let next_epoch_info = self.get_epoch_info(&next_epoch_id)?.clone();
-        let EpochInfoAggregator {
-            block_tracker: block_validator_tracker,
-            shard_tracker: chunk_validator_tracker,
-            all_proposals,
-            version_tracker,
-            ..
-        } = self.get_and_update_epoch_info_aggregator(
-            last_block_info.epoch_id(),
-            last_block_hash,
-            false,
-        )?;
+
+        self.update_epoch_info_aggregator(last_block_info.epoch_id(), last_block_hash, None)?;
```

Comment:
Similarly, here we can just use &self.epoch_info_aggregator directly. This is because this function is called https://github.com/near/nearcore/blob/master/chain/epoch_manager/src/lib.rs#L641, right after we just updated epoch_info_aggregator with the latest block